generator/generator_tool/generator_tool.cpp

```diff
@@ -122,6 +123,7 @@ DEFINE_string(srtm_path, "",
               "Path to srtm directory. If set, generates a section with altitude information "
               "about roads.");
 DEFINE_string(transit_path, "", "Path to directory with transit graphs in json.");
+DEFINE_bool(make_city_roads, false, "Makes section with feature ids lies inside cities");
```

Comment:
Calculates which features lie inside cities and makes a section with ids of these features.

Reply:
Done